app/pages/organizations/organization-page.jsx

```diff
@@ -186,33 +243,10 @@ class OrganizationPage extends React.Component {
             </div>
           </div>
 
-          <div className="organization-page__container">
-            <div className="project-metadata">
-              <span className="organization-details__heading">
-                {this.props.organization.display_name}{' '}<Translate content="project.home.metadata.statistics" />
-              </span>
-              <div className="project-metadata-stats">
-                <div className="project-metadata-stat">
-                  <div className="project-metadata-stat__value">{123}</div>
-                  <div className="project-metadata-stat__label">
-                    <Translate content="project.home.metadata.volunteers" />
-                  </div>
-                </div>
-                <div className="project-metadata-stat">
-                  <div className="project-metadata-stat__value">{456}</div>
-                  <div className="project-metadata-stat__label">
-                    <Translate content="project.home.metadata.classifications" />
-                  </div>
-                </div>
-                <div className="project-metadata-stat">
-                  <div className="project-metadata-stat__value">{789}</div>
-                  <div className="project-metadata-stat__label">
-                    <Translate content="project.home.metadata.subjects" />
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
+          <OrganizationMetaData
+            displayName={this.props.organization.display_name}
```

Comment:
Also, why snake case for `display_name` throughout this file? Nothing else in organization-page is snake case.